routes/users.js

```diff
@@ -437,7 +437,11 @@ router.post('/edit_order', function(req, res, next) {
             var vend = order.vendors[i];
             if (vendID.toString() === vend.vendID.toString()) {
               newQuantity = quantity - vend.quantity;
-              await UserHelper.addToCart(req.session.userId, ingID, newQuantity, vendor);
+              if (newQuantity <= -vend.quantity) {
```

Comment:
I still don’t understand why you can’t use newQuantity<=0

Reply:
Because the newQuantity will be negative if the user wants to make the quantity smaller